arkouda_metrics_exporter/README.md

```diff
@@ -2,16 +2,16 @@
 
 The arkouda_metrics_exporter project provides a Prometheus metrics export capability to Arkouda via the ArkoudaMetrics class, which does the following:
 
-1. Polls the Arkouda metrics socket for metrics
-2. Updates corresponding Prometheus data structures
+1. Polls the Arkouda metrics socket for metrics generated in the [MetricsMsg module](https://github.com/Bears-R-Us/arkouda/blob/master/src/MetricsMsg.chpl)
+2. Updates corresponding Prometheus data structures cached in the ArkoudaMetrics object
 3. Provides a Prometheus scrape target
 
 The arkouda_metrics_exporter project leverages the Prometheus [prometheus-client](https://github.com/prometheus/client_python) for all Prometheus metrics as well as the embedded http_server to deliver the Prometheus scrape target.
 
 
 ## Installation
 
-arkouda_metrics_exporter depends upon [arkouda](https://github.com/Bears-R-Us/arkouda) and [prometheus_client](https://pypi.org/project/prometheus-client/). Since there currently is no pypi Arkouda installer, an arkouda clone/code download followed by pip install precedes installing arkouda_metrics_exporter:
+arkouda_metrics_exporter depends upon [Arkouda](https://github.com/Bears-R-Us/arkouda) and [prometheus_client](https://pypi.org/project/prometheus-client/). Since there currently is no pypi Arkouda installer, an arkouda clone/code download followed by pip install precedes installing arkouda_metrics_exporter:
```

Comment:
Should we point to the `INSTALL.md` in the `arkouda` repo for installation instructions?

Reply:
@Ethan-DeBandi99 good point, I will update now

Reply:
@Ethan-DeBandi99 done!